ocaml/runtime4/caml/prims.h

```diff
@@ -26,9 +26,7 @@ extern c_primitive caml_builtin_cprim[];
 extern char * caml_names_of_builtin_cprim[];
 
 extern struct ext_table caml_prim_table;
-#ifdef DEBUG
```

Comment:
We should probably have the same in `ocaml/runtime`.

Reply:
Actually I'm not sure this change is required, let's try without.